nautobot_ssot/exceptions.py

```diff
@@ -0,0 +1,67 @@
+"""Custom Exceptions to be used with SSoT integrations."""
+
+
+class AdapterLoadException(Exception):
+    """Raised when there's an error while loading data."""
+
+
+class AuthFailure(Exception):
+    """Exception raised when authenticating to on-prem CVP fails."""
+
+    def __init__(self, error_code, message):
+        """Populate exception information."""
+        self.expression = error_code
```

Comment:
What does this do?

```py
self.expression = error_code
```

Reply:
I'm not too sure, this was done as a consolidation of all the custom Exceptions that I found in the various integrations. This particular one was in Arista CVP.